src/TraceEvent/TraceEvent.Tests/EventPipeTestBase.cs

```diff
@@ -18,7 +18,8 @@ protected EventPipeTestBase(ITestOutputHelper output)
         }
 
         private static IEnumerable<string> TestEventPipeZipFiles
-            => Directory.EnumerateFiles(TestDataDir, "*.netperf.zip");
+            => Directory.EnumerateFiles(TestDataDir, "*.netperf.zip")
+            .Union(Directory.EnumerateFiles(TestDataDir, "*.nettrace.zip"));
```

Comment:
Thank you for adding a new test case!  It looks like the baseline.txt file for the new test is empty, so something went wrong.

Reply:
Well that very conveniently explains why it never passed in CI : )

Reply:
Debugging it I realize part of what happened. Locally I worked around a bug in TraceEvent where the ContentionData for the ClrParser is out-of-sync with the event definition in the runtime. I just hand-editted it to make it match because I think you said hand-editting was necessary in an email a while back, but let me know if that wasn't the right way to fix this.

With that out of the way now I am checking in a shiny new baseline.

Reply:
Awesome.  Yes, hand editing is required at this point.